src/vueWrapper.ts

```diff
@@ -38,7 +50,124 @@ export class VueWrapper<T extends ComponentPublicInstance> {
     return this.vm.$el.parentElement
   }
 
+  /**
+   * Get all the target elements defined in the teleportTarget mount option.
+   *
+   * @private
+   *
+   * @return {Element[]|undefined}
+   */
+  private getTeleportElements(): Element[] | undefined | never {
+    if (!this.teleportTargets.length) {
+      return
+    }
+
+    const elements: Element[] = []
+
+    this.teleportTargets.forEach((target) => {
+      if (typeof target !== 'string') {
+        elements.push(target)
+        return
+      }
+
+      const element = document.querySelector(target)
+
+      if (!element) {
+        throw new Error("The teleport target '" + target + "' cannot be found.")
+      }
+
+      elements.push(element)
+    })
+
+    return elements
+  }
+
+  /**
+   * Get the teleport target element for the given vue instance.
+   *
+   * @param {ComponentPublicInstance} vm
+   *
+   * @private
+   *
+   * @return {Element}
+   */
+  private getTeleportElement(
+    vm: ComponentPublicInstance = this.componentVM
+  ): Element | never {
+    const elements = this.getTeleportElements()
+    const errorMsg =
+      vm.$.type.name +
+      "'s teleport target element cannot be found. Are you sure the target exits in the DOM?"
+
+    if (!elements || !elements.length) {
```

Comment:
`!elements` can be removed if `getTeleportElements` returns `Element[]` as suggested above.